lua/wikis/valorant/MatchPage.lua

```diff
@@ -73,7 +78,92 @@ end
 ---@param game MatchPageGame
 ---@return Widget
 function MatchPage:renderGame(game)
-	return HtmlWidgets.Fragment{}
+	return HtmlWidgets.Fragment{
+		children = WidgetUtil.collect(
+			self:_renderPerformance(game)
+		)
+	}
```

Comment:
technically we do not need the collect, right?
(if more stuff gets added it might be needed but with current code it seems redundant)

Reply:
More stuff will be added